e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/types/ConstraintResult.java

```diff
@@ -5,9 +5,7 @@
 import javax.json.JsonString;
 import java.util.List;
 
-public record ConstraintResult(int constraintId,
-                        String constraintName,
-                        String type,
+public record ConstraintResult(String type,
```

Comment:
Minor, ignorable: Formatting inconsistency:
```suggestion
public record ConstraintResult(
         String type,
```

Reply:
Formatting should be fixed